lib/lotus/generators/action.rb

```diff
@@ -48,22 +48,23 @@ def start
           template_path: _template_path,
         }
 
+        case options[:test]
+        when 'rspec'
+          test_type = 'rspec'
+        else
+          test_type = 'minitest'
+        end
+
         templates = {
           'action.rb.tt' => _action_path,
-          'view.rb.tt'   => _view_path,
-          'template.tt'  => _template_path
+          'action_spec.' + test_type + '.tt' => _action_spec_path,
```

Comment:
What do you think about string interpolation which help prevent exception when `test_type` is nil. that is `"action_spec.#{test_type}.tt"`